addons/mail/demo/discuss_channel_demo.xml

```diff
@@ -233,5 +233,39 @@
             <field name="seen_message_id" ref="mail.troubleshooting_message_2"/>
             <field name="new_message_separator" eval="ref('mail.troubleshooting_message_2') + 1"/>
         </record>
+        <record model="mail.message" id="mail.channel_public_message_0">
+            <field name="model">discuss.channel</field>
+            <field name="res_id" ref="mail.channel_public"/>
+            <field name="message_type">comment</field>
+            <field name="subtype_id" ref="mail.mt_comment"/>
+            <field name="author_id" ref="base.partner_admin"/>
+            <field name="body">We're excited to announce our product launch this September! 🎉
+            We'll be hosting a live video conference on this channel to showcase all the details.
+            If you're interested, feel free to share the link with your customers so they can tune
+            in and watch it live!</field>
+        </record>
+        <record model="mail.message" id="mail.channel_public_message_1">
+            <field name="model">discuss.channel</field>
+            <field name="res_id" ref="mail.channel_public"/>
+            <field name="message_type">comment</field>
+            <field name="subtype_id" ref="mail.mt_comment"/>
+            <field name="author_id" ref="base.partner_demo"/>
+            <field name="body">That sounds amazing! Can't wait to see what's new.
+            Will there be a QA session during the conference?</field>
+        </record>
+        <record model="discuss.channel.member" id="mail.admin_member_channel_public">
+            <field name="partner_id" ref="base.partner_admin"/>
+            <field name="channel_id" ref="mail.channel_public"/>
+            <field name="fetched_message_id" ref="mail.channel_public_message_0"/>
+            <field name="seen_message_id" ref="mail.channel_public_message_0"/>
+            <field name="new_message_separator" eval="ref('mail.channel_public_message_0') + 1"/>
+        </record>
+        <record model="discuss.channel.member" id="mail.demo_member_channel_public">
+            <field name="partner_id" ref="base.partner_demo"/>
+            <field name="channel_id" ref="mail.channel_public"/>
+            <field name="fetched_message_id" ref="mail.channel_public_message_1"/>
+            <field name="seen_message_id" ref="mail.channel_public_message_1"/>
+            <field name="new_message_separator" eval="ref('mail.channel_public_message_1') + 1"/>
+        </record>
```

Comment:
It would be nice to have a message from a guest and a guest as member too while we are at it.